pyproject.toml

```diff
@@ -106,6 +106,9 @@ datasets = [
     # xarray 0.12.3+ required for pandas 1.3.3 support
     "xarray>=0.12.3",
 ]
+models = [
```

Comment:
@adamjstewart I added this new extras group but not sure if I'm missing something else in the repo that I need to update as well.